Lib/test/test_capi/test_abstract.py

```diff
@@ -1000,6 +1000,12 @@ def test_number_check(self):
         self.assertTrue(number_check(0.5))
         self.assertFalse(number_check("1 + 1j"))
 
+    def test_object_generichash(self):
+        # Test PyObject_GenericHash()
+        generichash = _testcapi.object_generichash
+        for obj in object(), 1, 'string', []:
+            self.assertEqual(generichash(obj), object.__hash__(obj))
```

Comment:
Maybe test also that it's equal to hash(id(obj))?

Reply:
It is an implementation detail. It should be tested whether `object.__hash__` is tested, if tested at all.